media-gfx/icoutils/icoutils-0.31.1.recipe

```diff
@@ -0,0 +1,73 @@
+# FIXME: extresso and genresscript won't work because of missing libwww-perl
+# and that won't compile because reasons
+SUMMARY="Tools for extracting and converting Windows icon and cursor files"
+DESCRIPTION="A set of command-line programs for extracting and converting \
+images in Microsoft Windows icon and cursor files (usually with the extension \
+.ico or .cur, but they can also be embedded in executables and DLL libraries."
+HOMEPAGE="http://nongnu.org/icoutils/"
+COPYRIGHT="1998-2017 Frank Richter"
+LICENSE="GNU GPL v3"
+REVISION="1"
+SOURCE_URI="http://savannah.nongnu.org/download/icoutils/icoutils-$portVersion.tar.bz2"
+CHECKSUM_SHA256="26e29d3c78f25d4cdf402501ac0414c51a9a092daebf6c9dee3b837dee693093"
+PATCHES="icoutils-$portVersion.patchset"
+
+ARCHITECTURES="x86_gcc2 x86 x86_64"
+
+PROVIDES="
+	icoutil$secondaryArchSuffix = $portVersion
```

Comment:
I'm not so sure about the `PATCH()` function. I already generated a patchfile and the scripts don't work anyway because of missing LWP, so I don't think the way we change the hashbang matters all that much.